jsk_arc2017_baxter/euslisp/lib/arc-interface.l

```diff
@@ -333,6 +333,21 @@
             :rpy (float-vector 0 0 0))
           :rotation-axis rotation-axis
           :use-gripper use-gripper))
+  (:ik->tote-center
+    (arm &key (offset #f(0 0 0)) (rpy #f(0 0 0)) (rotation-axis t) (use-gripper nil))
+    (let (tote-cube tote-coords)
+      (setq tote-cube (gethash arm tote-cubes-))
+      (setq tote-coords (send tote-cube :copy-worldcoords))
+      (send tote-coords :rotate (aref rpy 0) :z)
+      (send tote-coords :rotate (aref rpy 1) :y)
+      (send tote-coords :rotate (aref rpy 2) :x)
+      (send tote-coords :translate
+            (v+ (float-vector
+                  0.0 0.0 (/ (z-of-cube tote-cube) 2.0))
+                 offset) :world)
```

Comment:
"z" of `z-of-cube` is repesented in local coodinates, isn't it?
Then please translate in local.
Offset can be applied in whether local or world.